backend/ipmi.pm

```diff
@@ -147,7 +147,9 @@ sub get_mc_status {
 
     $self->ipmitool("mc guid");
     $self->ipmitool("mc info");
-    $self->ipmitool("mc selftest");
+    if (!$bmwqemu::vars{IPMI_SKIP_SELFTEST}) {
+        $self->ipmitool("mc selftest");
+    }
```

Comment:
please make that `$self->ipmitool("mc selftest") unless $bmwqemu::vars{IPMI_SKIP_SELFTEST}`

Reply:
Yeah sure.